test/e2e/e2e.go

```diff
@@ -1659,6 +1659,58 @@ var _ = ginkgo.Describe("e2e ingress gateway traffic validation", func() {
 	})
 })
 
+// This test perform a longevity test, where it spawns multiple container
+// and wait for them to finish executing sleep. It does this exercise for
+// 20 minutes and stops. Each container starts, sleeps for 10 seconds and
+// goes to success or failure state.
+var _ = ginkgo.Describe("e2e longevity test", func() {
+	var svcname = "nettest"
+
+	f := framework.NewDefaultFramework(svcname)
+
+	ginkgo.BeforeEach(func() {
+
+		// retrieve worker node names
+		nodes, err := e2enode.GetBoundedReadySchedulableNodes(f.ClientSet, 3)
+		framework.ExpectNoError(err)
+		if len(nodes.Items) < 3 {
+			framework.Failf(
+				"Test requires >= 3 Ready nodes, but there are only %v nodes",
+				len(nodes.Items))
+		}
+	})
+
+	ginkgo.It("should create pods for a while without failing", func() {
+		ginkgo.By("Continuously creating pods for a while")
+
+		pod := 0
+		startTime := time.Now()
+
+		for time.Since(startTime) < 20*time.Minute {
+			// spin up 10 containers that each do nothing for 30 seconds
+
+			var errChans [10]chan error
+			for i, _ := range errChans {
+				errChans[i] = make(chan error)
+				go checkPodSleep(f, "", fmt.Sprintf("test-container-%v", pod), errChans[i])
+				pod++
+
+				time.Sleep(500 * time.Millisecond)
```

Comment:
what's this for?

Reply:
@jluhrsen sleep time between pod creation.

Reply:
I know :)

I should have ask why you have it.

Reply:
Just to avoid choking github low spec VM with sudden cpu/mem spike. :)